src/core/webview/__tests__/ClineProvider.test.ts

```diff
@@ -227,12 +227,6 @@ jest.mock("../../../integrations/misc/extract-text", () => ({
 	}),
 }))
 
-// Spy on console.error and console.log to suppress expected messages
-beforeAll(() => {
```

Comment:
I wasted some time trying to figure out why my `console.log` debugging wasn't working these tests. This was a bad idea 😬